editor/themes/editor_theme_manager.cpp

```diff
@@ -0,0 +1,2232 @@
+		p_theme->set_constant("h_separation", "HFlowContainer", p_config.separation_margin);
+		p_theme->set_constant("v_separation", "HFlowContainer", p_config.separation_margin);
+		p_theme->set_constant("h_separation", "VFlowContainer", p_config.separation_margin);
+		p_theme->set_constant("v_separation", "VFlowContainer", p_config.separation_margin);
+
+		// SplitContainer.
+
+		p_theme->set_icon("h_grabber", "SplitContainer", p_theme->get_icon(SNAME("GuiHsplitter"), EditorStringName(EditorIcons)));
+		p_theme->set_icon("v_grabber", "SplitContainer", p_theme->get_icon(SNAME("GuiVsplitter"), EditorStringName(EditorIcons)));
+		p_theme->set_icon("grabber", "VSplitContainer", p_theme->get_icon(SNAME("GuiVsplitter"), EditorStringName(EditorIcons)));
+		p_theme->set_icon("grabber", "HSplitContainer", p_theme->get_icon(SNAME("GuiHsplitter"), EditorStringName(EditorIcons)));
+
+		p_theme->set_constant("separation", "SplitContainer", p_config.separation_margin);
+		p_theme->set_constant("separation", "HSplitContainer", p_config.separation_margin);
+		p_theme->set_constant("separation", "VSplitContainer", p_config.separation_margin);
+
+		p_theme->set_constant("minimum_grab_thickness", "SplitContainer", p_config.increased_margin * EDSCALE);
+		p_theme->set_constant("minimum_grab_thickness", "HSplitContainer", p_config.increased_margin * EDSCALE);
+		p_theme->set_constant("minimum_grab_thickness", "VSplitContainer", p_config.increased_margin * EDSCALE);
+
+		// GridContainer.
+		p_theme->set_constant("v_separation", "GridContainer", Math::round(p_config.widget_margin.y - 2 * EDSCALE));
+	}
+
+	// Window and dialogs.
+	{
+		// Window.
+
+		p_theme->set_stylebox("embedded_border", "Window", p_config.window_style);
+		p_theme->set_stylebox("embedded_unfocused_border", "Window", p_config.window_style);
+
+		p_theme->set_color("title_color", "Window", p_config.font_color);
+		p_theme->set_icon("close", "Window", p_theme->get_icon(SNAME("GuiClose"), EditorStringName(EditorIcons)));
+		p_theme->set_icon("close_pressed", "Window", p_theme->get_icon(SNAME("GuiClose"), EditorStringName(EditorIcons)));
+		p_theme->set_constant("close_h_offset", "Window", 22 * EDSCALE);
+		p_theme->set_constant("close_v_offset", "Window", 20 * EDSCALE);
+		p_theme->set_constant("title_height", "Window", 24 * EDSCALE);
+		p_theme->set_constant("resize_margin", "Window", 4 * EDSCALE);
+		p_theme->set_font("title_font", "Window", p_theme->get_font(SNAME("title"), EditorStringName(EditorFonts)));
+		p_theme->set_font_size("title_font_size", "Window", p_theme->get_font_size(SNAME("title_size"), EditorStringName(EditorFonts)));
+
+		// AcceptDialog.
+		p_theme->set_stylebox("panel", "AcceptDialog", p_config.dialog_style);
+		p_theme->set_constant("buttons_separation", "AcceptDialog", 8 * EDSCALE);
+
+		// FileDialog.
+		p_theme->set_icon("folder", "FileDialog", p_theme->get_icon(SNAME("Folder"), EditorStringName(EditorIcons)));
```

Comment:
Inconsistent use of SNAME. This line has 4 StringNames, but 2 of them use raw strings. Though we have such cases everywhere in the codebase :/

Reply:
Yeah, we can do a cleanup here. I also wanted to add `get_editor_icon` to `EditorTheme` to avoid all the repetition.